usagov/usagov-2021
# USAGOV-1848-Move-Carousel-JS-and-CSS-out-of-Twig

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1848

## Description

Move scripts and styles into drupal libraries

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Visit the homepage and confirm that the life events carousel is working. 
Test the page load performance to confirm that the carousel assets dont delay the rendering of the page.
On the Dev environment:
 - Confirm that the life events carousel works on the static site.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
